src/main/target/NOX/target.h

```diff
@@ -44,9 +44,11 @@
 #define GYRO_1_CS_PIN           PB12
 #define GYRO_1_SPI_INSTANCE     SPI2
 
-#define USE_EXTI
-#define USE_GYRO_EXTI
-#define GYRO_1_EXTI_PIN         PA8
+// temporarily disable until bugfix is found
+//#define USE_EXTI
+//#define USE_GYRO_EXTI
+//#define GYRO_1_EXTI_PIN         PA8
```

Comment:
I suggest just commenting out line 49 in the interests of being concise and matching https://github.com/betaflight/unified-targets/pull/542

Reply:
Done!